configs/kempner_dev/7b_Olmo.yaml

```diff
@@ -0,0 +1,155 @@
+run_name: OLMo-7B
+seed: 6198
+dry_run: false
+
+wandb:
+  name: ${run_name}
+  project: olmo-medium
+  group: OLMo-7B
+
+model:
+  d_model: 4096
+  n_heads: 32
+  n_layers: 32
+  mlp_hidden_size: 16384 #22016
+  weight_tying: false
+  alibi: false
+  rope: true
+  flash_attention: false
+  attention_dropout: 0.0
+  attention_layer_norm: true
+  multi_query_attention: false
+  include_bias: false
+  block_type: sequential
+  layer_norm_type: default
+  layer_norm_with_affine: true
+  bias_for_layer_norm: false
+  attention_layer_norm_with_affine: true
+  activation_type: gelu
+  residual_dropout: 0.0
+  embedding_dropout: 0.0
+  max_sequence_length: 2048
+  vocab_size: 32100
```

Comment:
Is this vocab size correct?

Reply:
This is taken from their official config for 7b.

Reply:
the official 7b config has vocab_size: 50280

Reply:
No, it will not run. The vocab size is dictated by the tokenizer you are using. You have changed the tokenizer from the official config because we are using a different tokenizer for our data. I didn't use the same one, so I was asking if you double-checked to make sure 32100 is right. I looked through your code, and I think it is right for t5 base.

Reply:
That is correct, it is for `t5-base` tokenizer.

Reply:
Then it should be as it is.